tests/CMakeLists.txt

```diff
@@ -162,6 +163,10 @@ set_property(TEST nodeos_repeat_transaction_lr_test PROPERTY LABELS long_running
 
 add_test(NAME cli_test COMMAND tests/cli_test.py WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 
+add_test(NAME plugin_http_api_test COMMAND tests/plugin_http_api_test.py -v --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+set_tests_properties(plugin_http_api_test PROPERTIES TIMEOUT 3000)
+set_property(TEST plugin_http_api_test PROPERTY LABELS plugin_http_api_tests)
```

Comment:
Applying this label will create a new test group called `plugin_http_api_tests`, which the CI system will have to be modified to know to look for. Instead of creating a new test group, please add your test to one of the existing test groups:
- No label for unit/parallelizable tests
- `nonparallelizable_tests` for tests that run fast (<7 min) but have conflicts with other tests (ports, files, PIDs, etc) and cannot be parallelized
- `long_running_tests` for tests that take a long time to run
As a general rule of thumb, new tests should never be the longest-running test in their test group.

Reply:
set as "nonparallelizable_tests"